### PR TITLE
3box implemented and with proper styling

### PR DESCRIPTION
**Pull request by bconfortin**

### CODE REVIEW

**Comment by justinbarry** [comment]
I keep seeing this locally. Will try to rope someone else into trying it.
image

**Comment by justinbarry** [comment]
@bconfortin Should I be seeing this widget when logged in with portis?

**Comment by bconfortin** [comment]
You have to enable 3box in order to test it.

Go to `packages/augur-artifacts/src/environments/local.json` and add the following lines of code to that json:
```json
"plugins": {
  "comments": "3box"
}
```

File: packages/augur-ui/src/modules/market/components/common/comments/market-comments.tsx [hunk 1]
```diff
@@ -1,160 +1,103 @@
...
+          box={box}
+          currentUserAddr={address}
+
+          // optional
+          showCommentCount={numPosts}
+          currentUser3BoxProfile={profile}
+          // useHovers={true}
+        />
+      )}
+      {whichCommentPlugin === 'facebook' && (
```
Comment (justinbarry) [inline packages/augur-ui/src/modules/market/components/common/comments/market-comments.tsx:93]: Should we just drop facebook now?

Comment (bconfortin) [reply packages/augur-ui/src/modules/market/components/common/comments/market-comments.tsx:93]: Paul asked me to leave it as a choice. The effort for styling and all is already done and it adds very little to the final bundle, I guess

**Review by justinbarry — APPROVED** [review]
Minor styling issue @bconfortin is aware of. Otherwise LGTM!

**Pull request closed without merging**